Production build fails due to esbuild postinstall issue if an http proxy is required

### Description of the bug

I'm getting the following error when building my Vaadin 23 app within a `openjdk:11` docker container using Gradle, if the usage of an http proxy is required.

```
> Task :webui:vaadinBuildFrontend FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':webui:vaadinBuildFrontend'.
> Webpack process exited with non-zero exit code.
  Stderr: '/usr/src/myapp/webui/node_modules/esbuild/bin/esbuild:2
  throw new Error(`esbuild: Failed to install correctly
  ^

  Error: esbuild: Failed to install correctly

  Make sure you don't have "ignore-scripts" set to true. You can check this with
  "npm config get ignore-scripts". If that returns true you can reset it back to
  false using "npm config set ignore-scripts false" and then reinstall esbuild.

  If you're using npm v7, make sure your package-lock.json file contains either
  "lockfileVersion": 1 or the code "hasInstallScript": true. If it doesn't have
  either of those, then it is likely the case that a known bug in npm v7 has
  corrupted your package-lock.json file. Regenerating your package-lock.json file
  should fix this issue.

      at Object.<anonymous> (/usr/src/myapp/webui/node_modules/esbuild/bin/esbuild:2:7)
      at Module._compile (node:internal/modules/cjs/loader:1103:14)
      at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
      at Module.load (node:internal/modules/cjs/loader:981:32)
      at Function.Module._load (node:internal/modules/cjs/loader:822:12)
      at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
      at node:internal/main/run_main_module:17:47
  Hash: 261d2660434a681f5f84
  Version: webpack 4.46.0
  Time: 6604ms
  Built at: 04/26/2022 12:54:33 PM
                                                     Asset      Size  Chunks               Chunk Names
  VAADIN/build/vaadin-bundle-7ee22721b009c497ca15.cache.js  1.42 KiB       0  [immutable]  bundle
  Entrypoint bundle = VAADIN/build/vaadin-bundle-7ee22721b009c497ca15.cache.js
  [0] ./generated/vaadin.ts 530 bytes {0} [built] [failed] [1 error]

  ERROR in ./generated/vaadin.ts
  Module build failed (from ../node_modules/esbuild-loader/dist/index.js):
  Error: The service was stopped
      at /usr/src/myapp/webui/node_modules/esbuild/lib/main.js:1258:29
      at /usr/src/myapp/webui/node_modules/esbuild/lib/main.js:611:9
      at Socket.afterClose (/usr/src/myapp/webui/node_modules/esbuild/lib/main.js:589:7)
      at Socket.emit (node:events:538:35)
      at endReadableNT (node:internal/streams/readable:1345:12)
      at processTicksAndRejections (node:internal/process/task_queues:83:21)
  Child HtmlWebpackCompiler:
                            Asset      Size  Chunks  Chunk Names
      __child-HtmlWebpackPlugin_0  4.33 KiB       0  HtmlWebpackPlugin_0
      Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
      [0] ../node_modules/html-webpack-plugin/lib/loader.js!./index.html 756 bytes {0} [built]
  <i> [build-status] 1 error and 0 warnings were reported.
  <i> [build-status] : Failed to compile.
  '

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 11m 17s
```

### Expected behavior

No errors. The issue seems to be how the postinstall script for `esbuild` is called via https://github.com/vaadin/flow/blob/dcac32fa4de8fcf0d45ed778d326f913c461f851/flow-server/src/main/java/com/vaadin/flow/server/frontend/TaskRunNpmInstall.java#L476-L485

The `http_proxy` / `https_proxy` env variables are not passed on the npm processs. This works, if I run the postscript [out of Gradle](https://github.com/chkpnt/vaadin-issue13624/blob/c5a8ba933abda9ddbcbbb30c036505b3e579281d/build.sh#L19-L27).

But it's weird, that the `npm run postinstall` process called by Vaadin's Gradle plugin doesn't respect an npm config located in `node_modules/esbuild/.npmrc`, `~/.npmrc` or `/etc/npmrc` as shown [here](https://github.com/chkpnt/vaadin-issue13624/blob/c5a8ba933abda9ddbcbbb30c036505b3e579281d/build.sh#L12-L17).

### Minimal reproducible example

see https://github.com/chkpnt/vaadin-issue13624

Just run `docker compose up --build`

In case you want to reproduce this issue directly on your development machine without docker, you need to ensure that a proxy is needed and configured and there is no esbuild binary cached. See [`getCachePath(...)`](https://github.com/evanw/esbuild/blob/7d15c6d3d3115b173dfe20d74e5bdcf546fe9118/lib/npm/install.ts#L95-L107) for the caching location.

### Versions

- Vaadin / Flow version: 23.0.7
- Java version: 11
- OS version: Docker image based on `openjdk:11`
- Browser version (if applicable):
- Application Server (if applicable):
- IDE (if applicable):

Mh, I've just executed `npm install` within the docker container. It seems to be a permission issue:

```
npm info run esbuild@0.12.29 postinstall node_modules/esbuild node install.js
npm info run esbuild@0.12.29 postinstall { code: 1, signal: null }
npm WARN cleanup Failed to remove some directories [
npm WARN cleanup   [
npm WARN cleanup     undefined,
npm WARN cleanup     AssertionError [ERR_ASSERTION]: rimraf: missing path
npm WARN cleanup         at rimraf (/usr/share/nodejs/rimraf/rimraf.js:54:3)
npm WARN cleanup         at internal/util.js:297:30
npm WARN cleanup         at new Promise (<anonymous>)
npm WARN cleanup         at rimraf (internal/util.js:296:12)
npm WARN cleanup         at /usr/share/nodejs/@npmcli/arborist/lib/arborist/reify.js:394:20
npm WARN cleanup         at Array.map (<anonymous>)
npm WARN cleanup         at Arborist.[rollbackCreateSparseTree] (/usr/share/nodejs/@npmcli/arborist/lib/arborist/reify.js:394:8)
npm WARN cleanup         at /usr/share/nodejs/@npmcli/arborist/lib/arborist/reify.js:801:50
npm WARN cleanup         at runMicrotasks (<anonymous>)
npm WARN cleanup         at processTicksAndRejections (internal/process/task_queues.js:97:5) {
npm WARN cleanup       generatedMessage: false,
npm WARN cleanup       code: 'ERR_ASSERTION',
npm WARN cleanup       actual: undefined,
npm WARN cleanup       expected: true,
npm WARN cleanup       operator: '=='
npm WARN cleanup     }
npm WARN cleanup   ]
npm WARN cleanup ]
npm timing reify:rollback:createSparse Completed in 35821ms
npm timing reify:rollback:retireShallow Completed in 1ms
npm timing command:ci Completed in 628520ms
npm verb stack Error: command failed
npm verb stack     at ChildProcess.<anonymous> (/usr/share/nodejs/@npmcli/promise-spawn/index.js:64:27)
npm verb stack     at ChildProcess.emit (events.js:314:20)
npm verb stack     at maybeClose (internal/child_process.js:1022:16)
npm verb stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
npm verb pkgid esbuild@0.12.29
npm verb cwd /usr/src/myapp/webui
npm verb Linux 5.3.18-150300.59.54-default
npm verb argv "/usr/bin/node" "/usr/bin/npm" "ci" "--verbose"
npm verb node v12.22.5
npm verb npm  v7.5.2
npm ERR! code 1
npm ERR! path /usr/src/myapp/webui/node_modules/esbuild
npm ERR! command failed
npm ERR! command sh -c node install.js
npm ERR! Trying to install "esbuild-linux-64" using npm
npm ERR! Failed to install "esbuild-linux-64" using npm: Command failed: npm install --loglevel=error --prefer-offline --no-audit --progress=false esbuild-linux-64@0.12.29
npm ERR! glob error [Error: EACCES: permission denied, scandir '/root/.npm/_logs'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'scandir',
npm ERR!   path: '/root/.npm/_logs'
npm ERR! }
npm ERR! npm ERR! code EACCES
npm ERR! npm ERR! syscall open
npm ERR! npm ERR! path /root/.npm/_cacache/index-v5/64/0c/9be850d6d8812efb46dc50671f3853a0699ca70fa5fd35a645a18ca9f5fb
npm ERR! npm ERR! errno -13
npm ERR! npm ERR!
npm ERR! npm ERR! Your cache folder contains root-owned files, due to a bug in
npm ERR! npm ERR! previous versions of npm which has since been addressed.
npm ERR! npm ERR!
npm ERR! npm ERR! To permanently fix this problem, please run:
npm ERR! npm ERR!   sudo chown -R 1000:1000 "/root/.npm"
npm ERR!
npm ERR! Trying to download "https://registry.npmjs.org/esbuild-linux-64/-/esbuild-linux-64-0.12.29.tgz"
npm ERR! Failed to download "https://registry.npmjs.org/esbuild-linux-64/-/esbuild-linux-64-0.12.29.tgz": connect ETIMEDOUT 104.16.25.35:443
npm ERR! Install unsuccessful
npm verb exit 1
npm timing npm Completed in 628990ms

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-04-26T13_15_17_573Z-debug.log
```

And esbuild seems to ignore the proxy defined in `.npmrc`.

Yes, the postinstall script for `esbuild` is executed. But it doesn't seem to do anything but silently timeout. Around 130 seconds later, the second postinstall for `@vaadin/vaadin-usage-statistics` is called.

<details>
<summary>Extract from the log file</summary>

```
vaadin-issue13624-openjdk-1  | 2022-04-28T21:21:53.952+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:21:53.952+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] > Task :vaadinBuildFrontend
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.553+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdatePackages] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.553+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdatePackages] added 1186 packages in 1m
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.553+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdatePackages] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.553+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdatePackages] 95 packages are looking for funding
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.553+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdatePackages]   run `npm fund` for details
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.620+0000 [INFO] [com.vaadin.flow.server.frontend.TaskUpdatePackages] Frontend dependencies resolved successfully.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.622+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdatePackages] Running postinstall for 'esbuild'
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:06.305+0000 [LIFECYCLE] [org.gradle.cache.internal.DefaultFileLockManager] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:06.305+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:06.305+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:06.306+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:06.306+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:06.307+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:06.307+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:16.302+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:16.303+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:16.303+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:16.304+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:16.304+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:16.304+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:26.302+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:26.302+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:26.305+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:26.306+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:26.306+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:26.306+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:36.269+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:36.270+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:36.271+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:36.271+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:36.271+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:36.272+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:46.269+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:46.269+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:46.269+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:46.270+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:46.270+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:46.271+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:56.271+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:56.271+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:56.272+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:56.272+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:56.273+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:56.273+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:06.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:06.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:06.235+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:06.235+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:06.236+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:06.236+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:16.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:16.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:16.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:16.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:16.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:16.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:26.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:26.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:26.234+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:26.235+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:26.235+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:26.235+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:36.201+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:36.202+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:36.202+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:36.203+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:36.203+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:36.204+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:46.199+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:46.200+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:46.200+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:46.201+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:46.201+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:46.201+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:56.201+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:56.201+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:56.201+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:56.202+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:56.202+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:24:56.202+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:06.166+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:06.167+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:06.167+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:06.168+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:06.169+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:06.169+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.654+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:23:01.654+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] > Task :vaadinBuildFrontend
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:12.771+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdatePackages] Running postinstall for '@vaadin/vaadin-usage-statistics'
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.532+0000 [INFO] [com.vaadin.flow.server.frontend.TaskCopyFrontendFiles] Copying frontend resources from jar files ...
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.637+0000 [INFO] [com.vaadin.flow.server.frontend.TaskCopyFrontendFiles] Visited 104 resources. Took 104 ms.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.638+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskCopyLocalFrontendFiles] Found no local frontend resources for the project
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.689+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdateImports] Updated /app/build/frontend/generated-flow-imports-fallback.js
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.761+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdateImports] Updated /app/build/frontend/generated-flow-imports.js
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.761+0000 [DEBUG] [com.vaadin.flow.server.frontend.TaskUpdateImports] Updated /app/build/frontend/generated-flow-imports.d.ts
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.770+0000 [LIFECYCLE] [org.gradle.internal.operations.DefaultBuildOperationRunner] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.770+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Resolve files of :runtimeClasspath' started
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.770+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Resolve files of :runtimeClasspath'
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.719+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.719+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] > Task :vaadinBuildFrontend
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:13.770+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Resolve files of :runtimeClasspath' completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:14.904+0000 [INFO] [org.reflections.Reflections] Reflections took 1123 ms to scan 105 urls, producing 4578 keys and 23250 values
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:14.914+0000 [INFO] [org.gradle.api.Project] Passing this classpath to NodeTasks.Builder: [spring-boot-devtools-2.2.4.RELEASE.jar, vaadin-spring-boot-starter-23.0.7.jar, spring-boot-starter-web-2.2.4.RELEASE.jar, spring-boot-starter-json-2.2.4.RELEASE.jar, spring-boot-starter-validation-2.2.4.RELEASE.jar, spring-boot-starter-2.2.4.RELEASE.jar, spring-boot-autoconfigure-2.2.4.RELEASE.jar, spring-boot-2.2.4.RELEASE.jar, vaadin-spring-23.0.5.jar, vaadin-core-23.0.7.jar, spring-webmvc-5.2.3.RELEASE.jar, spring-websocket-5.2.3.RELEASE.jar, spring-context-5.2.3.RELEASE.jar, spring-web-5.2.3.RELEASE.jar, spring-aop-5.2.3.RELEASE.jar, spring-beans-5.2.3.RELEASE.jar, spring-expression-5.2.3.RELEASE.jar, spring-core-5.2.3.RELEASE.jar, vaadin-dev-server-23.0.5.jar, flow-lit-template-23.0.5.jar, flow-polymer-template-23.0.5.jar, flow-html-components-23.0.5.jar, flow-data-23.0.5.jar, flow-dnd-23.0.5.jar, flow-server-23.0.5.jar, flow-push-23.0.5.jar, flow-client-23.0.5.jar, hibernate-validator-6.0.18.Final.jar, vaadin-lumo-theme-23.0.7.jar, vaadin-material-theme-23.0.7.jar, vaadin-accordion-flow-23.0.7.jar, vaadin-avatar-flow-23.0.7.jar, vaadin-app-layout-flow-23.0.7.jar, vaadin-button-flow-23.0.7.jar, vaadin-checkbox-flow-23.0.7.jar, vaadin-combo-box-flow-23.0.7.jar, vaadin-custom-field-flow-23.0.7.jar, vaadin-date-time-picker-flow-23.0.7.jar, vaadin-date-picker-flow-23.0.7.jar, vaadin-details-flow-23.0.7.jar, vaadin-time-picker-flow-23.0.7.jar, vaadin-select-flow-23.0.7.jar, vaadin-dialog-flow-23.0.7.jar, vaadin-form-layout-flow-23.0.7.jar, vaadin-grid-flow-23.0.7.jar, vaadin-icons-flow-23.0.7.jar, vaadin-iron-list-flow-23.0.7.jar, vaadin-virtual-list-flow-23.0.7.jar, vaadin-list-box-flow-23.0.7.jar, vaadin-login-flow-23.0.7.jar, vaadin-messages-flow-23.0.7.jar, vaadin-ordered-layout-flow-23.0.7.jar, vaadin-progress-bar-flow-23.0.7.jar, vaadin-radio-button-flow-23.0.7.jar, vaadin-notification-flow-23.0.7.jar, vaadin-menu-bar-flow-23.0.7.jar, vaadin-context-menu-flow-23.0.7.jar, vaadin-renderer-flow-23.0.7.jar, vaadin-split-layout-flow-23.0.7.jar, vaadin-tabs-flow-23.0.7.jar, vaadin-text-field-flow-23.0.7.jar, vaadin-upload-flow-23.0.7.jar, spring-boot-starter-tomcat-2.2.4.RELEASE.jar, spring-jcl-5.2.3.RELEASE.jar, ph-css-6.5.0.jar, atmosphere-runtime-2.7.3.slf4jvaadin4.jar, ph-commons-10.1.6.jar, spring-boot-starter-logging-2.2.4.RELEASE.jar, logback-classic-1.2.3.jar, log4j-to-slf4j-2.12.1.jar, jul-to-slf4j-1.7.30.jar, slf4j-api-1.7.32.jar, javax.annotation-api-1.3.2.jar, gwt-elemental-2.8.2.vaadin2.jar, commons-fileupload-1.4.jar, commons-io-2.11.0.jar, jackson-datatype-jdk8-2.10.2.jar, jackson-datatype-jsr310-2.10.2.jar, jackson-module-parameter-names-2.10.2.jar, jackson-databind-2.10.2.jar, jackson-core-2.10.2.jar, jsoup-1.14.3.jar, byte-buddy-1.10.6.jar, gentyref-1.2.0.vaadin1.jar, commons-compress-1.21.jar, httpclient-4.5.10.jar, commons-codec-1.13.jar, validation-api-2.0.1.Final.jar, jboss-logging-3.4.1.Final.jar, classmate-1.5.1.jar, vaadin__vaadin-mobile-drag-drop-1.0.1.jar, mobile-drag-drop-2.3.0-rc.2.jar, commons-lang3-3.9.jar, jakarta.annotation-api-1.3.5.jar, snakeyaml-1.25.jar, tomcat-embed-websocket-9.0.30.jar, tomcat-embed-core-9.0.30.jar, tomcat-embed-el-9.0.30.jar, jakarta.validation-api-2.0.2.jar, jackson-annotations-2.10.2.jar, httpcore-4.4.13.jar, jsr305-3.0.2.jar, logback-core-1.2.3.jar, log4j-api-2.12.1.jar, /app/build/classes/java/main]
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:15.088+0000 [INFO] [com.vaadin.flow.server.frontend.FrontendTools] Globally node version 12.22.5 is older than 16.14.0. Using node from /root/.vaadin.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:15.108+0000 [DEBUG] [com.vaadin.flow.server.frontend.FrontendTools] Using node 16.14.2 located at /root/.vaadin/node/node
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:15.867+0000 [DEBUG] [com.vaadin.flow.server.frontend.FrontendTools] Using npm 8.5.0 located at /root/.vaadin/node/node
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.003+0000 [INFO] [org.gradle.api.Project] Running Webpack ...
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.003+0000 [DEBUG] [org.gradle.api.Project] 
vaadin-issue13624-openjdk-1  | /root/.vaadin/node/node \ 
vaadin-issue13624-openjdk-1  |     ./node_modules/webpack/bin/webpack.js 
vaadin-issue13624-openjdk-1  | 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.006+0000 [DEBUG] [org.zeroturnaround.exec.ProcessExecutor] Executing [/root/.vaadin/node/node, /app/node_modules/webpack/bin/webpack.js] in /app with environment {PATH=/root/.vaadin/node:/usr/local/openjdk-11/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin, http_proxy=http://squid:3128, https_proxy=http://squid:3128, JAVA_HOME=/usr/local/openjdk-11, OLDPWD=/app, LANG=C.UTF-8, HOSTNAME=1c6cc96703d5, JAVA_VERSION=11.0.15, PWD=/app, SHLVL=1, HOME=/root, _=./gradlew}.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.008+0000 [DEBUG] [org.zeroturnaround.exec.ProcessExecutor] Started Process[pid=257, exitValue="not exited"]
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.166+0000 [LIFECYCLE] [org.gradle.cache.internal.DefaultFileLockManager] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.166+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.166+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.166+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.167+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Waiting to acquire shared lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.167+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Lock acquired on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:16.167+0000 [DEBUG] [org.gradle.cache.internal.DefaultFileLockManager] Releasing lock on daemon addresses registry.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.917+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Execute vaadinBuildFrontend for :vaadinBuildFrontend'
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.920+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Executing task ':vaadinBuildFrontend''
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:15.919+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:15.919+0000 [LIFECYCLE] [class org.gradle.internal.buildevents.TaskExecutionLogger] > Task :vaadinBuildFrontend FAILED
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.911+0000 [DEBUG] [org.zeroturnaround.exec.WaitForProcess] Process[pid=257, exitValue=1] stopped with exit code 1
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.917+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Execute vaadinBuildFrontend for :vaadinBuildFrontend' completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.920+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Executing task ':vaadinBuildFrontend'' completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.924+0000 [DEBUG] [org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter] Removed task artifact state for task ':vaadinBuildFrontend' from context.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.925+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Task :vaadinBuildFrontend'
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.925+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Task :vaadinBuildFrontend' completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.925+0000 [INFO] [org.gradle.execution.plan.DefaultPlanExecutor] :vaadinBuildFrontend (Thread[Execution worker for ':',5,main]) completed. Took 3 mins 34.348 secs.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.925+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :vaadinBuildFrontend failed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :bootJar completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :assemble
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.Node] Dependency :jar for :assemble not yet completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :jar completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :assemble
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.Node] All dependencies are complete for :assemble
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all must successors are complete for :assemble
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all finalizing successors are complete for :assemble
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] All task dependencies are complete for :assemble
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :assemble completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :build
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.Node] Dependency :check for :build not yet completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :compileTestJava completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.926+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :testClasses
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.Node] Dependency :processTestResources for :testClasses not yet completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :processTestResources completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :testClasses
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.Node] All dependencies are complete for :testClasses
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all must successors are complete for :testClasses
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all finalizing successors are complete for :testClasses
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] All task dependencies are complete for :testClasses
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :testClasses completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :test
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.Node] All dependencies are complete for :test
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all must successors are complete for :test
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all finalizing successors are complete for :test
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] All task dependencies are complete for :test
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :test completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :check
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.Node] All dependencies are complete for :check
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all must successors are complete for :check
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all finalizing successors are complete for :check
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.927+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] All task dependencies are complete for :check
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :check completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :build
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.Node] All dependencies are complete for :build
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all must successors are complete for :build
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all finalizing successors are complete for :build
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] All task dependencies are complete for :build
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :build completed, executed: false
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.DefaultExecutionPlan] Node :vaadinBuildFrontend completed, executed: true
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :jar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.Node] All dependencies are complete for :jar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all must successors are complete for :jar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all finalizing successors are complete for :jar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] All task dependencies are complete for :jar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.Node] Checking if all dependencies are complete for :bootJar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.Node] All dependencies are complete for :bootJar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all must successors are complete for :bootJar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] Checking if all finalizing successors are complete for :bootJar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.928+0000 [DEBUG] [org.gradle.execution.plan.TaskNode] All task dependencies are complete for :bootJar
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.929+0000 [DEBUG] [org.gradle.internal.resources.AbstractTrackedResourceLock] Execution worker for ':': released lock on :
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.929+0000 [DEBUG] [org.gradle.internal.work.DefaultWorkerLeaseService] Worker lease root.1.110 completed (1 worker(s) in use)
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.929+0000 [DEBUG] [org.gradle.internal.resources.AbstractTrackedResourceLock] Execution worker for ':': released lock on root.1.110
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.929+0000 [DEBUG] [org.gradle.execution.plan.DefaultPlanExecutor] Execution worker [Thread[Daemon worker,5,main]] finished, busy: 32.337 secs, idle: 3 mins 44.493 secs
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.930+0000 [DEBUG] [org.gradle.execution.plan.DefaultPlanExecutor] Execution worker [Thread[Execution worker for ':',5,main]] finished, busy: 3 mins 44.418 secs, idle: 32.401 secs
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.931+0000 [DEBUG] [org.gradle.execution.taskgraph.DefaultTaskExecutionGraph] Timing: Executing the DAG took 4 mins 16.835 secs
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.945+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Run tasks'
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.945+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Run tasks' completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.947+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Run main tasks'
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.947+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Run main tasks' completed
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.985+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.986+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.986+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.986+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.987+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Execution failed for task ':vaadinBuildFrontend'.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.987+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Webpack process exited with non-zero exit code.
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.987+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   Stderr: '/app/node_modules/esbuild/bin/esbuild:2
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.987+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   throw new Error(`esbuild: Failed to install correctly
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.987+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   ^
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.987+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   
vaadin-issue13624-openjdk-1  | 2022-04-28T21:25:23.987+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]   Error: esbuild: Failed to install correctly
...
```
</details>

--- a/flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java
+++ b/flow-server/src/main/java/com/vaadin/flow/server/frontend/NodeUpdater.java
@@ -396,7 +396,7 @@ public abstract class NodeUpdater implements FallibleCommand {
             defaults.put("workbox-build", WORKBOX_VERSION);
         } else {
             // Webpack plugins and helpers
-            defaults.put("esbuild-loader", "2.15.1");
+            defaults.put("esbuild-loader", "2.19.0");
             defaults.put("html-webpack-plugin", "4.5.1");
             defaults.put("fork-ts-checker-webpack-plugin", "6.2.1");
             defaults.put("webpack", "4.46.0");

